### ISSUE TITLE
[Android] UI Tests are not using https images

### ISSUE THREAD

**Issue by jeromelaban** [unoplatform/uno#1597]
## Current behavior

The UI Tests samples are using many http:// unsecure Uris, and this is against the new Android 9. defaults. 

## Expected behavior

All samples use external https:// images, and the related samples are restored. 

## Environment

Nuget Package:

Package Version(s): 2.0-dev.x

Affected platform(s):
- [ ] iOS
- [x] Android
- [ ] WebAssembly
- [ ] Windows
- [ ] Build tasks

Visual Studio
- [ ] 2017 (version: )
- [ ] 2019 (version: )
- [ ] for Mac (version: )

Relevant plugins
- [ ] Resharper (version: )

**Comment by gagandeepp** [comment]
@jeromelaban assign this to me

**Comment by jeromelaban** [comment]
Thank you for your help!

**Comment by gagandeepp** [comment]
how to test this? Do I need to run Uno.UI.Tests ???

**Comment by jeromelaban** [comment]
@gagandeepp yes, but you can also rely on the CI output to determine if the Urls work properly.

**Comment by nishp77** [comment]
@jeromelaban anyone working on it??

**Comment by jeromelaban** [comment]
@nishp77 not at this point, you can have a stab at it.

**Comment by nishp77** [comment]
@jeromelaban well had wasted lot of time setting up as I use rider not VS and use linux on top it, I want to confirm that UI.Test samples, do you mean sample apps for android?

**Comment by jeromelaban** [comment]
This issue refers to the samples app that is included in the Uno solution, here's [an example](https://github.com/unoplatform/uno/blob/629558362a44d76cc62496045ab714e7ed485f5f/src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/ImageTests/Models/ImageWithLateSourceViewModel.cs#L24).

**Comment by nishp77** [comment]
What if some sites are not using https? do we remove them?

**Comment by jeromelaban** [comment]
Most of the times, those images are just tests, so replacing with something else like https://picsum.photos/ for now

**Comment by p3rid0t** [comment]
Seems very easy to fix, please assign this issue to me

**Comment by MartinZikmund** [comment]
Not currently planned, if interest remains, can be revisited
